server/api/gateKeepingMiddleware.js

```diff
@@ -0,0 +1,22 @@
+const express = require('express');
+const router = express.Router();
+const {
+  models: { User },
+} = require('../db');
+
+// MIDDLEWARE
+router.use(express.json());
```

Comment:
Line 8 -- don't think this is required here. i see this middleware is already included in /server/app.js line 11

Reply:
Good catch. Just deleted it. Will recommit.